helm/cluster-gcp/templates/_control_plane.tpl

```diff
@@ -6,14 +6,15 @@ Any changes to this will trigger the resource to be recreated rather than attemp
 {{- define "controlplane-gcpmachinetemplate-spec" -}}
 image: {{ include "vmImage" $ }}
 instanceType: {{ .Values.controlPlane.instanceType }}
-rootDeviceSize: {{ .Values.controlPlane.rootVolumeSizeGB }}
+rootDeviceSize: {{ .Values.controlPlane.rootVolume.sizeGB | default 100 }}
+rootDeviceType: {{ .Values.controlPlane.rootVolume.diskType | default "pd-ssd" }}
```

Comment:
Out of curiosity, what did this default to before? I'm guessing `pd-ssd`?

Reply:
Before there was no default and therefore it was taking the default disk type coming from CAPG which is "pd-standard".
I recommend to change it to "pd-ssd" which is more perfomant